README.md

```diff
@@ -13,6 +13,8 @@ Teleport component
 | type       | Type of teleport: line or parabolic         | parabolic   |
 | button       | Button used to launch the teleport: trackpad, trigger, grip, menu         | trackpad   |
 | collisionEntities | Selector of the meshes used to check the collisions. If no value provided a plane Y=0 is used |  |
+| endEvents | Paired with `startEvents`, list of events to listen for to finish teleporting.| []            |
+| teleportStartEvents         | Alternative to `button`, list of events to listen to start teleporting.| []            |
```

Comment:
dont need this line

Reply:
thanks! I shouldn't close this PR that fast :D